src/bentoml/_internal/bento/default_svc_readme.py

```diff
@@ -0,0 +1,21 @@
+readme_template = """# {{ svc.name }}:{{ svc_version }}
```

Comment:
If we are allowing Jinja template like this, we probably want to use block for user to extends or overwrite the template easily.